prow/scripts/cluster-integration/kyma-aks-long-lasting.sh

```diff
@@ -334,6 +334,10 @@ EOF
 
 	KYMA_RESOURCES_DIR="${KYMA_SOURCES_DIR}/installation/resources"
 
+  # WORKAROUND: add gateway ip address do IstioOperator in installer-config-production.yaml.tpl (see: https://github.com/kyma-project/test-infra/issues/2792)
+  echo "#### WORKAROUND: add gateway ip address do IstioOperator in installer-config-production.yaml.tpl (see: https://github.com/kyma-project/test-infra/issues/2792)"
+  sed -i "/k8s:/a\            service:\n              loadBalancerIP: ${GATEWAY_IP_ADDRESS}\n              type: LoadBalancer" "${KYMA_RESOURCES_DIR}"/installer-config-production.yaml.tpl
```

Comment:
I don't like that this will break as soon as someone removes/adds a space. Maybe it would be easier to just use a regex for loadBalancerIP for an inline replacement.

Reply:
I don't know if we can add these parameters as permanent to the file and change it inline in the script. The best way would be to use `yq` for adding `service` map to the yaml with those two parameters.